apps/advisor/pkg/app/checkscheduler/checkscheduler.go

```diff
@@ -214,6 +214,8 @@ func (r *Runner) cleanupChecks(ctx context.Context, logger logging.Logger) error
 		return err
 	}
 
+	logger.Debug("Cleaning up checks", "numChecks", len(list.GetItems()))
```

Comment:
These changes are unrelated, just wanted to have more debug logs in this function.